### PR TITLE
Feature/view

### PR DESCRIPTION
**Pull request by Larkgronick**
Request date:
31/03/21

Total:
• Fixed all meeting remarks 
• Changed JSX function calls
• Refactored contants objects
• Fixed styles issues (add low opacity while loading tables and popups representation)
• Maded proper form validation

### CODE REVIEW

File: src/services/services.js [hunk 1]
```diff
@@ -41,7 +41,8 @@ export const registerNewUser = async (email, password) => {
     await firebase.auth().sendPasswordResetEmail(email);
     return response;
   } catch (error) {
-    throw new Error('Error with registration:', error);
+    console.log('Error with registration:', error);
```
Comment (maximsan) [inline src/services/services.js:44]: only use `console.error`

**Review by maximsan — APPROVED** [review]
Nice. U need one more approve and I will merge it.

**Review by YauhenKoipish — APPROVED** [review]

File: src/services/services.js (hunk 1 above)
Comment (Larkgronick) [reply src/services/services.js:44]: Changed here.

Comment (Larkgronick) [reply src/services/services.js:44]: Changed in services methods.

**Pull request merged**